<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompSetup</class>
 <widget class="QDialog" name="CompSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>598</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>510</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>세정장치</string>
   </property>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>가동/정지 : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="autoCompIntv">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTime">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>변경</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>가동간격(분) : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>자동/수동 : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="autoComp">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>자동</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="runComp">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>정지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="autoCompTime">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>가동시간(초) : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAuto">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>변경</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRun">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>변경</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIntv">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>변경</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>510</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>보수설정</string>
   </property>
   <widget class="QPushButton" name="btnRepair">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>67</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>변경</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>22</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>고장 설정</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>67</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>보수 설정</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMalf">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>17</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>변경</string>
    </property>
   </widget>
   <widget class="QLabel" name="repair">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>67</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>정상</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="malf">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>22</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>정상</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>510</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>FAN</string>
   </property>
   <widget class="QPushButton" name="btnFan">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>15</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>변경</string>
    </property>
   </widget>
   <widget class="QLabel" name="runFan">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>25</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>정지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelFan">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>가동/정지</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>510</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>ACK</string>
   </property>
   <widget class="QPushButton" name="btnAck">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>변경</string>
    </property>
   </widget>
   <widget class="QLabel" name="useAck">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>ACK</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>CompSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
